lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -384,8 +384,7 @@ namespace Js
         Var tag = JavascriptOperators::GetProperty(thisArgAsObject, PropertyIds::_symbolToStringTag, scriptContext); // Let tag be the result of Get(O, @@toStringTag).
 
         // 17. Return the String that is the result of concatenating "[object ", tag, and "]". 
-        if (tag != nullptr && JavascriptString::Is(tag))
-        {
+        auto buildToString = [&scriptContext](Var tag) {
```

Comment:
curious what's the expected output of Object.getOwnPropertyNames(Math)? should we have Symbol.toString output as well?